### ISSUE TITLE
Add uninstall support!

### ISSUE THREAD

**Issue by hedayat** [ubuntu-ir/traktor#84]
I've recently come across a number of users who have problems with traktor, and want to be able to remove it (actually, revert all its modifications) completely. 

I've seen that you've had an uninstall script for Debian, but it has been removed because 'users can just change proxy', but: 1. it is not enough to revert all changes, e.g. adding tor repositories in Ubuntu/Debian, 2. Maybe not all users know how to change the proxy (they might not know where you've changed proxy settings).

A better, longer term approach would be to provide a .deb package which can be cleanly removed, but I know that needs some fundamental changes to how traktor works (e.g. currently, you set it up for the 'current user', but a .deb should set it up for all users or at least let any user to complete its setup by running a commandd).

**Comment by GreatBahram** [comment]
I'm sure there is no need to provide new deb file for installing some packages that already have deb file. I believe there must be very clear document that help newbie user and pro user to learn how to use and how to configure tor and other proxy like privoxy and so on. This is an insult to just install some package and ok user now you're secure. 
If the maintainer write this script for Iranian user, there should be farsi document. 
Thanks by the way

**Comment by hedayat** [comment]
No, it is completely OK and quite normal. Everybody is free to learn all this, but there is no need to force everybody to do that. Anyway, this issue is not about questioning this project's basis. You can fill another issue for that!

**Comment by Sosha1996** [comment]
Dear Hedayat,
These changes have already been applied to the tractor but must be approved by Danial Behzadi.

You can see the changes on [my GitLab](https://gitlab.com/Sosha/traktor)

**Comment by amirwacker** [comment]
salam chetor mitonam traktor ro pak konam?
mikham khodam tor ro tor ro nasb va config konam.

**Comment by GreatBahram** [comment]
@hedayat  As it can be seen, people need more document instead of some application do all things for them. I think there must be at least on shell script for temporary disable all configuration that Traktor had done.

**Comment by hedayat** [comment]
So they should not come here in the first place. Anyway, anybody is welcome to create such documentation, why don't you do it yourslef?!
But it is completely unrelated here. Traktor is a specific project with a specific goal. Arguing that a project should not exist is completely irrelevant. 

Finally, please stop arguing with me. I'm NOT a traktor developer, nor a traktor user! I just saw an area for Traktor improvement, and suggested that to its developers. That's it. 

If people don't need Traktor, they'll stop using it. And if you think something else is needed, either create it yourself or find people who'll do it. That should be pretty easy.

**Comment by vahidajalluian** [comment]
This is not a so good work, since it does have no gui as the tor in windows. I wonder if it is possible to contact tor  distributor to build a gui for the platform in ubuntu. Of course there are some alternatives for tor as lantern, but the only problem is its limited free monthly high speed data!

**Comment by danialbehzadi** [comment]
Tor doesn't supposed to have a GUI. There is one for windows because the lack of functionality there.
If anyone needs a GUI for Tor in GNU, they should use Tor Browser instead,
